crates/core/src/mock/mock_starknet.rs

```diff
@@ -141,6 +148,40 @@ pub fn fixtures(methods: Vec<AvailableFixtures>) -> Vec<StarknetRpcFixture> {
         .collect()
 }
 
+/// Returns all the fixtures present in the fixtures directory.
+/// # Panics
+/// Returns an error if a fixture is not a valid json or if the fixture is not present in both
+/// requests and responses.
+///
+/// Panics if `all_fixtures` is called from a directory where `core` crate is not in scope.
+pub fn all_fixtures() -> Vec<StarknetRpcFixture> {
+    let cwd = &std::env::current_dir().unwrap();
+    let request_path = find_git_root_path(cwd).unwrap().join("crates/core/src/mock/fixtures/requests");
```

Comment:
would it change anything to mark `mock` as `cfg(test)`?

Reply:
not sure! since we use it as a test-util, we export it to other crates

Reply:
i can try

Reply:
actually now I understand why dojo has a dedicated util crate, so that they can build it properly and then only import it as dev dependency to keep a lean binary size for their business logic